### DISCUSSION TITLE
Adyen is now accepting only businesses with sales amounting to more than 5 million EUR/yearly

### DISCUSSION THREAD

**Discussion by iba-1** [saleor/saleor#9725] [General]
Hey @everyone,
we're building a saleor-based solution for one of our customers and we were planning to use Adyen + Stripe (fallback) as PG.

The problem is: Adyen has decided to restrict its customer base by accepting only customers who sell more than 5 million EUR on their e-commerce yearly (which is really a lot), so our customer was denied access to their Payment Gateway.
There is another condition that may allow a customer to sign up for Adyen: using one of their supported plug-in (https://docs.adyen.com/plugins) either built by them or in partnership with them.

Our understanding is that Adyen was one of the first pg implemented in Saleor infrastructure, is there any possibility that you already have a partnership relation with them and it's just not listed yet on that page, or that such relation could ever be instaurated?

Having such partnership with them could enable a lot of small to medium business to use saleor (out of the box, without having to extend both fe and be) even if they can't use Stripe.

We know it's a bit off topic maybe and more of a business question instead of a technical one, but we thought that there was room for discussion, so we tried 😃

**Comment by timuric** [comment]
Hi @iba-1 we are not aware of such Adyen limitations. We are going to release a new payment app and checkout this week which will feature Adyen and Mollie integrations (stay tuned). I would recommend checking out Mollie, which offers similar payment options and rates, as well as being friendly to small merchants.

**Reply by iba-1** [reply]
@timuric thank you for your reply, indeed we wrote you to make you aware of such limitations, which are 100% in place as we we're in touch with them and our customer was negated access by their sales team with the following reason (both in italian and translated):

> Per quanto riguarda LDP, mi sono presa il tempo per esaminare le informazioni che mi hai fornito sul vostro cliente e sfortunatamente in questo momento Adyen può supportare solo aziende con transazioni superiori a 5 milioni di euro all'anno o aziende che sono attualmente supportate da un [plug-in](https://docs.adyen.com/plugins) progettato da Adyen. 

> For what concerns LDP, I took some time to examine the information you provided about it and unfortunately at this time Adyen can only support businesses with a total transaction amount higher than 5 million EUR/yearly or businesses that use one of our [plug-in](https://docs.adyen.com/plugins).

Again, this could be a major blocking reason to adopt Saleor (at least in EU) as Adyen's plugin is the only one, with Stripe, handling SCA-ready transactions, both with 3DS v1 and v2 (if I understand correctly). 

On title EDIT:
We apologize beforehand but in the hope of gaining a little bit more traction on this issue and to inform everyone looking to integrate Adyen, we're editing the title to "Adyen is now accepting only businesses with sales amounting to more than 5 million EUR/yearly".

EDIT:
5 million/yearly is a limit that most e-commerce won't ever reach, and accrediting Saleor integration as a recognized plugin by Adyen could help us (and the community) overcome this limitation.
